### PR TITLE
Jekyll Templating & SEO

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: samhita-alla <aallasamhita@gmail.com>

-  Copied changes from `master` to `development` branch (added navbar.html to include the navbar-related code)
-  Created sitemap.xml file
-  Added meta description
-  Added meta tag to enable Google Search Console Access
-  Jekyll SEO Tag
-  Modified Navbar

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
-title: Flyte by Lyft
-description: A managed platform for executing distributed workflows at scale.
+title: Flyte 
+description: Flyte enables production-grade orchestration for machine learning workflows and data processing created to accelerate local workflows to production.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://www.flyte.org" # the base hostname & protocol for your site, e.g. http://example.com
 
+plugins:
+  - jekyll-seo-tag
+
```
Comment (kumare3) [inline _config.yml:8]: We have to change links to docs

File: _includes/footer.html [hunk 2]
```diff
@@ -1,32 +1,72 @@
...
       <div class="w-full sm:w-full md:w-1/2 lg:w-4/6 h-auto">
-          <div class="mb-6 sm:mb-8 lg:mb-10 font-mono font-bold text-3xl leading-tight">Ready To Take Flyte?</div>
-          <a href="{{ site.getstarted }}" class="btn bg-white text-purple-darkest hover:bg-pink hover:text-purple-black">Get Started</a>
+        <div
+          class="mb-6 sm:mb-8 lg:mb-10 font-mono font-bold text-3xl leading-tight"
+        >
+          Ready To Take Flyte?
+        </div>
+        <a
+          href="https://docs.flyte.org/en/latest/getting_started/first_run.html"
```
Comment (kumare3) [inline _includes/footer.html:11]: This was a template right?

File: _includes/footer.html [hunk 3]
```diff
@@ -1,32 +1,72 @@
...
-              <li class="my-2"><a class="link-white hover:border-purple-lightest" href="{{ site.documentation }}">Documentation</a></li>
-              <li class="my-2"><a class="link-white hover:border-purple-lightest" href="{{ site.github }}">Github</a></li>
-              <li class="my-2"><a class="link-white hover:border-purple-lightest" href="{{ site.stackoverflow }}">StackOverflow</a></li>
-          </ul>
+        <div class="mb-2 text-body font-bold uppercase">Site</div>
+        <ul class="list-reset text-base leading-normal">
+          <li class="my-2">
+            <a
+              class="link-white hover:border-purple-lightest"
+              href="https://flyte.readthedocs.io/en/latest/"
```
Comment (kumare3) [inline _includes/footer.html:22]: Let's just use docs.Flyte.org

File: sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  <url>
+    <loc>https://flyte.org/</loc>
+    <lastmod>2021-03-17</lastmod>
+    <changefreq>monthly</changefreq>
+    <priority>1.0</priority>
+  </url>
+  <url>
+    <loc>https://docs.flyte.org/projects/flytekit/en/latest/</loc>
+    <lastmod>2021-03-31</lastmod>
+    <changefreq>monthly</changefreq>
```
Comment (kumare3) [inline sitemap.xml:12]: Can we make this daily?

**Review by kumare3 — COMMENTED** [review]
Couple questions- seems you are building the site but you removed all the templates- is that intentional?. I am ok with, just want to know

File: _config.yml (hunk 1 above)
Comment (samhita-alla) [reply _config.yml:8]: Done.

File: _includes/footer.html (hunk 2 above)
Comment (samhita-alla) [reply _includes/footer.html:11]: It's a template. I just removed the first `</div>` line from the file and included it in the `default.html` file. `footer`, `header`, and `navbar` HTML pages should only have their respective code blocks. If there are additional `div`-related tags, I've moved them to the `default.html` file.

The remaining modifications are due to the code prettifier.

File: _includes/footer.html (hunk 3 above)
Comment (samhita-alla) [reply _includes/footer.html:22]: Done.

File: sitemap.xml (hunk 4 above)
Comment (samhita-alla) [reply sitemap.xml:12]: Sure. I auto-generated the sitemap.xml file. So it's populated as monthly.

**Comment by samhita-alla** [comment]
I've removed the `_site` directory as it isn't required to be pushed to GitHub. Here's an excerpt from the documentation:

> _site: This is where the generated site will be placed (by default) once Jekyll is done transforming it. It’s probably a good idea to add this to your .gitignore file. 


I've also removed the `destination: _site` line from the `_config.yml` file as the same value is set by default.

**Review by kumare3 — APPROVED** [review]

**Pull request merged**